packages/core/src/external/aws/lambda-logic/bundle-to-html-derm.ts

```diff
@@ -2686,3 +2620,15 @@ function createChartInScript({
     }
   `;
 }
+
+function isDateAfter(date: string | undefined): boolean {
```

Comment:
Nice that we're building this function! But can we name it to indicate what's it after from?

Maybe `isWithinLastTwoYears()`?